etna/models/utils.py

```diff
@@ -50,3 +53,48 @@ def determine_num_steps(start_timestamp: pd.Timestamp, end_timestamp: pd.Timesta
             raise ValueError(f"End timestamp isn't reachable with freq: {freq}")
         cur_value += 1
         cur_timestamp = timestamps[-1]
+
+
+def select_observations(
+    df: pd.DataFrame,
+    timestamps: pd.Series,
+    freq: str,
+    start: Optional[Union[pd.Timestamp, str]] = None,
+    end: Optional[Union[pd.Timestamp, str]] = None,
+    periods: Optional[int] = None,
+) -> pd.DataFrame:
+    """Select observations from dataframe with known timeline.
+
+    Parameters
+    ----------
+    df:
+        dataframe with known timeline. If `timestamps` column is not presented `freq`, `start`,
```

Comment:
When can it happens that there is not timestamp column in df?

Reply:
In fact in all the cases we don't have timestamp column, may you simplify this method for our cases

Reply:
It is general purpose function that can be used in other places. Timeline could be provided within `df` or by function parameters.

Reply:
In case we need this function somewhere we can enhance it, but now it just make it overcomplicated